composer.json

```diff
@@ -22,17 +22,17 @@
     "minimum-stability": "dev",
     "require": {
         "php": ">=7.1.3",
```

Comment:
You can probably bump to 7.2 as Symfony 4.4 requires it. This would allow you to remove a CI job too :)

Reply:
I would love to - but I think 4.4 only requires 7.1 - https://github.com/symfony/skeleton/blob/1d596be50b63df6e8fdf94f89f7f38ae6d17e024/composer.json#L7

Reply:
Ah indeed!